lib/rpcxdr/tarpc_base.x.m4

```diff
@@ -891,6 +899,13 @@ struct tarpc_statvfs_out {
     struct tarpc_statvfs buf;
 };
 
+struct tarpc_readlink_out {
+    struct tarpc_out_arg common;
+
+    char                resolved_path<>;
```

Comment:
it should be `string<>`, not `char<>`, these are pretty different things in XDR, and while the latter is used extensively throughout legacy code, it's still an error.